app/views/course/assessment/submission/submissions/edit.html.slim

```diff
@@ -9,6 +9,8 @@
     h4 = t('.files')
     = render partial: 'layouts/materials', locals: { folder: @assessment.folder }
 
+= render 'progress' if can?(:grade, @submission)
```

Comment:
Fixed the `can(:grade)` ability to include submissions in the `attempting` state.